nodeJS/logs/error.log

```diff
@@ -0,0 +1,7 @@
+{"message":"400 - ID is not valid - /news/77 - GET - ::1","level":"error","timestamp":"2019-11-27 16:03:29"}
```

Comment:
Please, add this file to gitignore.

Reply:
I've added as a working example. Removed now =)